README.adoc

```diff
@@ -150,11 +158,11 @@ kabanero      nodejs-express           0.2.5      *simple, skaffold   Express we
 kabanero      nodejs-loopback          0.1.4      *scaffold           LoopBack 4 API Framework for Node.js
 ----
 
-In the output you can see multiple stacks (IDs) in the `kabanero` collection hub. Each collection includes a version number,
+In the output you can see multiple collections (IDs) in the `kabanero` Collection Hub. Each collection includes a version number,
 one or more templates (the asterisk (*****) indicates the default template), and a description.
```

Comment:
why are there 5 asterisks displayed in brackets after the word asterisk?

Reply:
Not sure where these came from. I've modified to say 

.. (an asterisk (*) indicates the default template) ...